keras_hub/src/utils/preset_utils.py

```diff
@@ -735,5 +689,93 @@ def load_preprocessor(
             return super().load_preprocessor(cls, **kwargs)
         # We found a `preprocessing.json` with a complete config for our class.
         preprocessor = load_serialized_object(preprocessor_json, **kwargs)
-        preprocessor.load_preset_assets(self.preset)
+        if hasattr(preprocessor, "load_preset_assets"):
+            preprocessor.load_preset_assets(self.preset)
         return preprocessor
+
+
+class KerasPresetSaver:
```

Comment:
We might want to clean this up in the future, for now I'm just consolidating this to match the loading helper we have.